### ISSUE TITLE
[Intl] Inappropriate English name for the "Czech Republic"

### ISSUE THREAD

**Issue by vvasiloi** [symfony/symfony#26246]
| Q                | A
| ---------------- | -----
| Bug report?      | **yes**
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 3.4.4

https://github.com/symfony/symfony/blob/5220b6f9fcdee261c6fdd3e096f3b204fba5d906/src/Symfony/Component/Intl/Resources/data/regions/en.json#L59

The "Czech Republic" is the official formal name while "Czechia" is the short name, so isn't it more appropriate to use the first one?
Source: https://en.wikipedia.org/wiki/Name_of_the_Czech_Republic

**Comment by fabpot** [comment]
Data come from ICU, so the bug should be reported there instead.

**Comment by stof** [comment]
To be exact, this data is the CLDR data, which is then included in ICU and taken from ICU to included it in Symfony. So please report this to the CLDR project: http://cldr.unicode.org/

**Comment by vvasiloi** [comment]
@stof I did that right after Fabian's answer: https://unicode.org/cldr/trac/ticket/10945

TL;DR: 
> We use customary names, not formal names.

Is there a way to extend/override the data of Symfony Intl Component?

**Comment by jakzal** [comment]
@vvasiloi no, there's no way to do that. We simply read what ICU provides. You'll need to handle it on your side.
